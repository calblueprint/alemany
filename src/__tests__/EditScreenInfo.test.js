import React from 'react';

import { cleanup, render } from '@testing-library/react-native';

import EditScreenInfo from '../components/EditScreenInfo';

afterEach(cleanup);

describe('Edit Screen Info', () => {
  it('should show hello world', () => {
    const text = 'Hello World!';

    const { toJSON, getByText } = render(<EditScreenInfo path={text} />);

    const elem = getByText(text);

    expect(elem.props.children).toEqual(text);
    expect(toJSON()).toMatchSnapshot();
  });
});
