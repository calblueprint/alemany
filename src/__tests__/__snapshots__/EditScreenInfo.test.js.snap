// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Edit Screen Info should show hello world 1`] = `
<View>
  <View
    style={
      Object {
        "alignItems": "center",
        "marginHorizontal": 50,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#000000",
            "fontFamily": "System",
            "fontWeight": "400",
          },
          Object {
            "textAlign": "left",
          },
          Object {
            "fontSize": 17,
            "lineHeight": 24,
            "textAlign": "center",
          },
        ]
      }
    >
      Open up the code for this screen:
    </Text>
    <View
      style={
        Array [
          Object {
            "borderRadius": 3,
            "paddingHorizontal": 4,
          },
          Object {
            "marginVertical": 7,
          },
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "System",
              "fontWeight": "400",
            },
            Object {
              "textAlign": "left",
            },
            Array [
              null,
              Object {
                "backgroundColor": "#fafafa",
                "fontFamily": "space-mono",
              },
            ],
          ]
        }
      >
        Hello World!
      </Text>
    </View>
    <Text
      style={
        Array [
          Object {
            "color": "#000000",
            "fontFamily": "System",
            "fontWeight": "400",
          },
          Object {
            "textAlign": "left",
          },
          Object {
            "fontSize": 17,
            "lineHeight": 24,
            "textAlign": "center",
          },
        ]
      }
    >
      Change any of the text, save the file, and your app will automatically update.
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "marginHorizontal": 20,
        "marginTop": 15,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
          "paddingVertical": 15,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "System",
              "fontWeight": "400",
            },
            Object {
              "textAlign": "left",
            },
            Object {
              "textAlign": "center",
            },
          ]
        }
      >
        Tap here if your app doesn't automatically update after making changes
      </Text>
    </View>
  </View>
</View>
`;
